opa-audit-operator/files/sync.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: config.gatekeeper.sh/v1alpha1
```

Comment:
Where is this file referenced? I see it commented out in the charm code. Can it be removed?

Reply:
Please remove this.

Reply:
:+1: